lib/ansible/galaxy/collection/__init__.py

```diff
@@ -616,12 +617,20 @@ def verify_collections(
                     )
                     if not os.path.isdir(b_search_path):
                         continue
+                    if not os.path.isfile(os.path.join(b_search_path, b'MANIFEST.json')):
```

Comment:
Could you import and use `_is_installed_collection_dir()` from `ansible.galaxy.dependency_resolution.dataclasses`?

Reply:
Done :+1: